cmd/litcli/ln.go

```diff
@@ -339,15 +360,20 @@ func sendPayment(ctx *cli.Context) error {
 			"is instead: %v", len(destNode))
 	}
 
+	rfqPeerKey, err := hex.DecodeString(ctx.String(rfqPeerPubKeyFlag.Name))
+	if err != nil {
+		return fmt.Errorf("unable to decode RFQ peer public key: "+
+			"%w", err)
+	}
```

Comment:
Does this work if the field is optionally unset?

Reply:
yes, decoding an empty string ("") is valid and doesn't return an error